ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -367,7 +381,15 @@ class ProfileMainScreen extends React.PureComponent<Props> {
               <React.Fragment>
                 {this.debugListItem(
                   `${I18n.t("profile.main.appVersion")} ${getAppLongVersion()}`,
-                  () => clipboardSetStringWithFeedback(getAppLongVersion()),
+                  () => {
+                    if (this.state.numberOfTaps === 4) {
+                      this.setState({ showPagoPAtestSwitch: true });
+                    } else {
+                      this.setState({
```

Comment:
This could cause an inconsistent updated: you are reading the state while you are updating it
Use instead:

```typescript
const numberOfTaps = this.state.numberOfTaps + 1;
this.setState({
  numberOfTaps: numberOfTaps
});
```